components.json

```diff
@@ -342,6 +342,10 @@
 			"title": "HCL",
 			"owner": "outsideris"
 		},
+		"nand2tetris-hdl": {
```

Comment:
One last thing: Please insert this at its alphanumerical correct position, after N4JS.

Reply:
Great catch. Done.